components/SlideIn/SlideIn.tsx

```diff
@@ -0,0 +1,61 @@
+import { Slide, SlideProps } from "@mui/material";
+import { FC, ReactElement, useEffect, useRef, useState } from "react";
+import { useInView } from "react-intersection-observer";
+
+interface ISlideIn extends SlideProps {
+	/**
+	 * Margin around the root.  Can have values similar to the CSS margin property, e.g. "10px 20px 30px 40px" (top, right, bottom, left).
```

Comment:
Great efforts to improve DX! Thanks

You can open up a new issue if you want to add such props comments in all global components. Please let me know if you are interested or not. If you are interested you can open up an issue.